### DISCUSSION TITLE
What is tree traversal?

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#38] [Q&A] [answered]
Brief

**Comment by Riturani87** [comment, marked as answer]
Tree traversal in a data structure is a type of graph traversal in the data structure that refers to the process of visiting, verifying, and updating each node in a tree data structure just once. The order in which you examine the nodes of the tree is used to classify these traversals.

**Comment by ankit-akash** [comment]
congrats!🎉You have recieved your Galaxy Brain Badge.
